<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerAttributeTableWidgetBase</class>
 <widget class="QWidget" name="QgsComposerAttributeTableWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attribute Table</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 2px; font-weight: bold; background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Attribute table</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>392</width>
        <height>1202</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Main properties</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mSourceComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mLayerLabel">
            <property name="text">
             <string>Layer</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsMapLayerComboBox" name="mLayerComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="mRelationLabel">
            <property name="text">
             <string>Relation</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="mRelationsComboBox"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="mRefreshPushButton">
            <property name="text">
             <string>Refresh table data</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="mAttributesPushButton">
            <property name="text">
             <string>Attributes...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox_5">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Feature filtering</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="mMaxNumFeaturesLabel">
            <property name="text">
             <string>Maximum rows</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mMaximumRowsSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsSpinBox" name="mMaximumRowsSpinBox">
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="showClearButton">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="mUniqueOnlyCheckBox">
            <property name="text">
             <string>Remove duplicate rows from table</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="mShowOnlyVisibleFeaturesCheckBox">
            <property name="text">
             <string>Show only features visible within a map</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="mComposerMapLabel">
            <property name="text">
             <string>Composer map</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mComposerMapComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="mComposerMapComboBox"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="mIntersectAtlasCheckBox">
            <property name="text">
             <string>Show only features intersecting atlas feature</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="mFeatureFilterCheckBox">
            <property name="text">
             <string>Filter with</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="mFeatureFilterEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="mFeatureFilterButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/images/themes/default/mIconExpression.svg</normaloff>:/images/themes/default/mIconExpression.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder>mShowOnlyVisibleFeaturesCheckBox</zorder>
         <zorder>mComposerMapLabel</zorder>
         <zorder>mComposerMapComboBox</zorder>
         <zorder>mFeatureFilterCheckBox</zorder>
         <zorder>mMaximumRowsSpinBox</zorder>
         <zorder>mMaxNumFeaturesLabel</zorder>
         <zorder>mIntersectAtlasCheckBox</zorder>
         <zorder>mUniqueOnlyCheckBox</zorder>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox_6">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Appearance</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="0">
           <widget class="QLabel" name="mMarginLabel">
            <property name="text">
             <string>Cell margins</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mMarginSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsDoubleSpinBox" name="mMarginSpinBox">
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="showClearButton">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Display header</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="mHeaderModeComboBox">
            <item>
             <property name="text">
              <string>On first frame</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>On all frames</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No header</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Empty tables</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="mEmptyModeComboBox"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="mEmptyMessageLabel">
            <property name="text">
             <string>Message to display</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="mEmptyMessageLineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Background color</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QgsColorButtonV2" name="mBackgroundColorButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="mDrawEmptyCheckBox">
            <property name="text">
             <string>Show empty rows</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mShowGridGroupCheckBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Show grid</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="mGridStrokeWidthLabel">
            <property name="text">
             <string>Stroke width</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mGridStrokeWidthSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsDoubleSpinBox" name="mGridStrokeWidthSpinBox">
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="showClearButton">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QgsColorButtonV2" name="mGridColorButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox_3">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Fonts and text styling</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Table heading</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Font</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="mHeaderFontPushButton">
               <property name="text">
                <string>Choose font...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QgsColorButtonV2" name="mHeaderFontColorButton">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Alignment</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="mHeaderHAlignmentComboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Follow column alignment</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Left</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Center</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Right</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Table contents</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="lblContentsFont">
               <property name="text">
                <string>Font</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="mContentFontPushButton">
               <property name="text">
                <string>Choose font...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QgsColorButtonV2" name="mContentFontColorButton">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="frameGroupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Frames</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="mResizeModeLabel">
            <property name="text">
             <string>Resize mode</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mResizeModeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="mAddFramePushButton">
            <property name="text">
             <string>Add Frame</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="mEmptyFrameCheckBox">
            <property name="text">
             <string>Don't export page if frame is empty</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="mHideEmptyBgCheckBox">
            <property name="text">
             <string>Don't draw background if frame is empty</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header location="global">qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>mSourceComboBox</tabstop>
  <tabstop>mLayerComboBox</tabstop>
  <tabstop>mRelationsComboBox</tabstop>
  <tabstop>mRefreshPushButton</tabstop>
  <tabstop>mAttributesPushButton</tabstop>
  <tabstop>groupBox_5</tabstop>
  <tabstop>mMaximumRowsSpinBox</tabstop>
  <tabstop>mUniqueOnlyCheckBox</tabstop>
  <tabstop>mShowOnlyVisibleFeaturesCheckBox</tabstop>
  <tabstop>mComposerMapComboBox</tabstop>
  <tabstop>mIntersectAtlasCheckBox</tabstop>
  <tabstop>mFeatureFilterCheckBox</tabstop>
  <tabstop>mFeatureFilterEdit</tabstop>
  <tabstop>mFeatureFilterButton</tabstop>
  <tabstop>groupBox_6</tabstop>
  <tabstop>mDrawEmptyCheckBox</tabstop>
  <tabstop>mMarginSpinBox</tabstop>
  <tabstop>mHeaderModeComboBox</tabstop>
  <tabstop>mEmptyModeComboBox</tabstop>
  <tabstop>mEmptyMessageLineEdit</tabstop>
  <tabstop>mBackgroundColorButton</tabstop>
  <tabstop>mShowGridGroupCheckBox</tabstop>
  <tabstop>mGridStrokeWidthSpinBox</tabstop>
  <tabstop>mGridColorButton</tabstop>
  <tabstop>groupBox_3</tabstop>
  <tabstop>mHeaderFontPushButton</tabstop>
  <tabstop>mHeaderFontColorButton</tabstop>
  <tabstop>mHeaderHAlignmentComboBox</tabstop>
  <tabstop>mContentFontPushButton</tabstop>
  <tabstop>mContentFontColorButton</tabstop>
  <tabstop>frameGroupBox</tabstop>
  <tabstop>mResizeModeComboBox</tabstop>
  <tabstop>mAddFramePushButton</tabstop>
  <tabstop>mEmptyFrameCheckBox</tabstop>
  <tabstop>mHideEmptyBgCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
